backend/spec/system/backoffice/auth_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "system_helper"
+
+RSpec.describe "Backoffice: Auth", type: :system do
+  let_it_be(:admin) { create(:admin, email: "admin@example.com", password: "SuperSecret6", first_name: "Admin", last_name: "Example") }
+
+  describe "Login" do
+    before { visit "/backoffice" }
+
+    it "authenticate user successfuly" do
+      fill_in :admin_email, with: "admin@example.com"
+      fill_in :admin_password, with: "SuperSecret6"
+
+      click_on "Sign in"
```

Comment:
just habit of mine, but used to put all these `fill_in` or `click_on` to `before` action and keep only expectations at `it` block

Reply:
I prefer to not write system specs in very BDD way, rspec style. Just to have a minimum number of `it` blocks. Better to add multiple expectation and test more in one block than have a rspec structure, because the specs will eventually be very slow if we add too many of them.